django/db/migrations/graph.py

```diff
@@ -156,7 +211,7 @@ def make_state(self, nodes=None, at_end=True, real_apps=None):
             nodes = list(self.leaf_nodes())
         if len(nodes) == 0:
             return ProjectState()
-        if not isinstance(nodes[0], tuple):
+        if not isinstance(nodes[0], (tuple, Node)):
```

Comment:
I don't understand how this should work with a `Node` instance. `forwards_plan()` would expect `node.key` I think.